### ISSUE TITLE
Saving a Doc2Vec model failed.error says that the model is not UTF-8 encoded

### ISSUE THREAD

**Issue by gharelim** [RaRe-Technologies/gensim#2287]
I am failing to save a trained Doc2Vec model that analyses Tweets.When i save it and try to open the content of the model it says that :saving is disabled and that the model is not UTF-8 encoded.Any idea about the error?
 **i create the model as follows:**
_model_ug_dbow = Doc2Vec(dm=0, size=100, negative=5, min_count=2, workers=2, alpha=0.065, min_alpha=0.065)_

**training:**

for epoch in range(30):
    model_ug_dbow.train(utils.shuffle([x for x in tqdm(all_x_w2v)]), total_examples=len(all_x_w2v), epochs=1)
    model_ug_dbow.alpha -= 0.002
    model_ug_dbow.min_alpha = model_ug_dbow.alpha

**Saving:**
_model_ug_dbow.save('d2v_model_ug_dbow.doc2vec')_

**Comment by menshikh-iv** [comment]
Hello @gharelim,

- You no need to control alpha manually, set `epochs=30` - model automatically do the same thing (if you use not an old version of gensim)
- Show exact stack trace that you get 

example below works correctly
```python
from gensim.test.utils import common_texts
from gensim.models.doc2vec import Doc2Vec, TaggedDocument

corpus = [TaggedDocument(doc, [idx]) for idx, doc in enumerate(common_texts)]

model = Doc2Vec(dm=0, size=100, negative=5, min_count=2, workers=2, alpha=0.065, min_alpha=0.065)
model.build_vocab(corpus)

model.train(corpus, total_examples=model.corpus_count, epochs=30)

model.save("/tmp/myd2v.doc2vec")
```
